project/FrontendBuild.scala

```diff
@@ -46,7 +46,7 @@ private object AppDependencies {
     def apply() = new TestDependencies {
       override lazy val test = Seq(
         "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % scope,
-        "uk.gov.hmrc"       %%    "hmrctest"   %  "3.5.0-play-25"  % scope,
+        "uk.gov.hmrc"       %%    "hmrctest"   %  "3.8.0-play-25"  % scope,
```

Comment:
Unless there is a direct dependency to a new / updated class this should ideally be done in a separate ticket

Reply:
This was just something i missed on a previous PR for dependency updates. I've reverted it now and will raise a separate PR for it